data/endpoints/actionsRouter.js

```diff
@@ -1,4 +1,92 @@
 const express = require('express');
 
-//importing actions
-const actions = require('../seeds/02-actions');
+//import router from express
+const router = express.Router();
+
+//importing actions database
+const ActionsDb = require('../helpers/actionModel');
+//importing projects database
+const ProjectsDb = require('../helpers/projectModel');
+
+//<------------------------------------------------------------------------- GET REQUESTS ----------------
```

Comment:
nice separation with comments and using them to describe whats being done in the sections as well